### ISSUE TITLE
Increase odds ratio resolution to 4 decimal points

### ISSUE THREAD

**Issue by d0choa** [opentargets/genetics#537]
### Context
As reported by @mohdkarim and @Jeremy37, for the next study-locus [link](https://genetics.devgen2202.opentargets.xyz/study-locus/GCST90038599/7_100691506_A_ATTTTTTT), we are displaying an odds ratio of 1 and CIs of (1.0, 1.0)

<img width="1768" alt="Screenshot 2022-02-22 at 11 32 36" src="https://user-images.githubusercontent.com/5097586/155124026-862427a9-5181-49a7-b94b-81bcf33d158c.png">
)

However, as it can be seen on the API response, we have more resolution for the odds ratio. The fact that the p-value is 3.0e-15 is telling us that the deviation between 1 and 1.0073 is relevant, so it would be good to be able to see the difference.

### Action

Can we please increase the resolution of the odds ratio and the confidence intervals to 4 decimal points?

This implies updating:
- The study locus page as shown in the previous screenshot
- The study page table as shown in the next screenshot for the same [study](https://genetics.devgen2202.opentargets.xyz/study/GCST90038599)

<img width="1462" alt="Screenshot 2022-02-22 at 11 39 53" src="https://user-images.githubusercontent.com/5097586/155125093-c91132ae-31f6-42e4-9320-624811980704.png">

**Comment by andrewhercules** [comment]
@d0choa there is a [single function called `significantFigures`](https://github.com/opentargets/genetics-app/blob/main/src/ot-ui-components/helpers/significantFigures.js) that uses the [`d3-format` library](https://github.com/d3/d3-format) to render all numbers found in Open Targets Genetics. 

Unfortunately, the current implementation does not provide granularity to specify a certain precision for odds ratio numbers and a different precision for L2G scores or p-values.

If I change the function to show 4 decimal points ([deploy preview](https://deploy-preview-206--genetics-app.netlify.app/study-locus/GCST90038599/7_100691506_A_ATTTTTTT)), this is what it would look like on a study-locus page.

![Screenshot 2022-02-23 at 14 15 38](https://user-images.githubusercontent.com/7490258/155337903-707fa496-df45-4691-afc3-054f9190af59.png)

And using the example you provided above -- [Diabetes or endocrine disease and locus around 7_100691506_A_ATTTTTTT](https://genetics.devgen2202.opentargets.xyz/study-locus/GCST90038599/7_100691506_A_ATTTTTTT) -- if I set the function to show 5 decimal points so that the odds ratio value of `1.0073` is visible, this is what it looks like.

![Screenshot 2022-02-23 at 14 15 20](https://user-images.githubusercontent.com/7490258/155337187-b7be138b-7ab8-4c97-a4f7-9918a0714c52.png)

Which option do you prefer?

**Comment by d0choa** [comment]
I see. The abstraction that assumes all figures will require the same resolution is probably not a good idea. Different numbers have different interpretations and might require different levels of resolution for reasons that are hard to generalise.

If @carcruz thinks it's the right strategy, we can create a second helper function `significantOdds` and apply it on odds ratios and their respective confidence intervals

**Comment by andrewhercules** [comment]
I agree, I don't think it is ideal that all numbers are treated the same as some numbers require different levels of precision. I have closed the corresponding PR - https://github.com/opentargets/genetics-app/pull/206 - as this issue requires a more comprehensive fix.
